pkg/api/api.go

```diff
@@ -22,3 +22,14 @@ const (
 	OpenShiftConsoleRouteName         = OpenShiftConsoleShortName
 	OAuthClientName                   = OpenShiftConsoleName
 )
+
+// configs always have the name "cluster"
+const (
+	ConsoleConfigName         = "cluster"
+	ConsoleOperatorConfigName = ConsoleConfigName
+)
+
+// however, the ClusterOperator resource is named "console"
+const (
+	ClusterOperatorName = "console"
```

Comment:
- `config.openshift.io`, `ClusterOperator`, name: `console`

Reply:
But the `namespace` for the `operator` is `openshift-console-operator`, and the `operator` `deployment` is named `console-operator`. 

Just tacking that on here for record :)